decidim-meetings/spec/system/meeting_questions_reply_poll_spec.rb

```diff
@@ -86,8 +87,10 @@
     end
 
     it "does not allow selecting two single options" do
-      expect(page).to have_css("details[data-question='#{question_single_option.id}']:not([disabled])")
+      expect(page).to have_css("details[data-question='#{question_single_option.id}']:not([disabled])", visible: :visible)
+      sleep(2)
```

Comment:
Same here

Reply:
The same [reason](https://github.com/decidim/decidim/pull/13028#discussion_r1652335768)